mosquitto/Chart.yaml

```
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.6"
 description: Eclipse Mosquitto is an open source message broker which implements MQTT version 5, 3.1.1 and 3.1
 name: mosquitto
-version: 0.1.1
+version: 0.1.2
```

As it's a feature minor version should be raised.
```suggestion
version: 0.2.0
```